### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by UltCombo**

### CODE REVIEW

File: spec.html [hunk 1]
```diff
@@ -31044,7 +31044,7 @@
             If _obj_ is an exotic object (including Proxy exotic objects) whose behaviour for [[Get]], [[Set]], [[Delete]], and [[GetOwnProperty]] is not the ordinary object implementation of these internal methods.
           </li>
           <li>
-            If any index index property of _obj_ whose name is a nonnegative integer less than _len_ is an accessor property or is a data property whose [[Writable]] attribute is *false*.
+            If any index property of _obj_ whose name is a nonnegative integer less than _len_ is an accessor property or is a data property whose [[Writable]] attribute is *false*.
```
Comment (ljharb) [inline spec.html]: i think also `s/nonnegative/non-negative` would be an improvement

Comment (UltCombo) [inline spec.html]: I think so too. Both seem to be valid, though. I'm not sure which is preferred.

Comment (domenic) [inline spec.html]: No, nonnegative is the mathematical term.

Comment (UltCombo) [inline spec.html]: Wolfram and Google's definitions use "nonnegative" as the canonical term. Wikipedia and Wiktionary seem to prefer "non-negative". I guess both terms are accepted, so there's no need to fix what is not broken.

Comment (ljharb) [inline spec.html]: From what I can see both are equally acceptable, in math or otherwise, so that suggests there's no reason to change it.

**Comment by bterlson** [comment]
Committed as b8ac48f. Thanks!

**Pull request closed without merging**